pkg/api/mlflow/config/config.go

```diff
@@ -42,25 +45,50 @@ func NewServiceConfig() *ServiceConfig {
 }
 
 // Validate validates service configuration.
-func (c ServiceConfig) Validate() error {
-	if err := c.validateArtifactRoot(); err != nil {
-		return eris.Wrap(err, "error validating service configuration")
+func (c *ServiceConfig) Validate() error {
+	if err := c.validateConfiguration(); err != nil {
+		return eris.Wrap(err, `error validating service configuration`)
+	}
+	if err := c.normaliseConfiguration(); err != nil {
+		return eris.Wrap(err, `error normalising service configuration`)
+	}
+	return nil
+}
+
+// validateConfiguration validates service configuration for correctness.
+func (c *ServiceConfig) validateConfiguration() error {
+	// 1. validate ArtifactRoot configuration parameter for correctness and valid values.
+	parsed, err := url.Parse(c.ArtifactRoot)
+	if err != nil {
+		return eris.Wrap(err, `error parsing "artifact-root" flag`)
```

Comment:
please use a consistent style for error messages — the flag/parameter is sometimes quoted with backticks(`` `schema` ``), single-quotes (`'path'`), or double-quotes (`"artifact-root"`) — I'd suggest sticking with single-quotes everywhere

Reply:
Ill double-check everything.

Reply:
this is still using backticks!
```suggestion
		return eris.Wrap(err, "error parsing 'artifact-root' flag")
```